jujugui/static/gui/src/app/app.js

```diff
@@ -703,42 +802,48 @@ YUI.add('juju-gui', function(Y) {
     },
 
     /**
+      Handles logging into both the env and controller api websockets.
+
+      @method loginToAPIs
+      @param {Object} credentials The credentials to pass to setCredentials.
+      @param {Boolean} useMacaroons If it should use the loginWithMacaroon
```

Comment:
useMacaroons Whether to use macaroon based auth (macaraq) or simple username/password auth?